test/server/util-test.js

```diff
@@ -72,10 +72,10 @@ describe('Utils', () => {
                 expected: {
                     name:            'Other',
                     version:         '0.0',
-                    platform:        'desktop',
-                    os:              { name: 'Windows', version: '0.0' },
+                    platform:        'other',
```

Comment:
This test seems unrelated to the changes made by @miherlosev. Is it caused by a `bowser` update? 

BTW, do we really need tests for invalid/incomplete user agents? 😕 @Farfurix

Reply:
Yes, it's change related to `bowser` module update.
Our tests check the default values for 'platform', 'version', etc.

Reply:
@AndreyBelym @miherlosev I'll check this case

Reply:
Now, I will merge this PR. Changes related to the tests will be done in the separate PR.